packages/lib/src/proxy/Proxy.ts

```diff
@@ -15,9 +15,9 @@ export default class Proxy {
   private txParams: any;
   public address: string;
 
-  public static at(address: string, txParams: any = {}): Proxy {
+  public static at(contractOrAddress: string | any, txParams: any = {}): Proxy {
```

Comment:
nit: both contract name and address are strings

Reply:
Actually not. This was quite tricky and something pointed out by @facuspagnuolo 

Proxy uses the following function from utils/Addresses.ts:

```
export function toAddress(contractOrAddress: string | any): string {
  if (_.isEmpty(contractOrAddress)) throw Error(`Contract or address expected`);
  else if (_.isString(contractOrAddress)) return contractOrAddress;
  else return contractOrAddress.address;
}
```